package.json

```diff
@@ -7,7 +7,10 @@
     "dev": "vite",
     "build": "vite build",
     "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "unit test": "vitest",
+    "unit report": "vitest --reporter=html",
```

Comment:
I added a command to generate a report. Do you think the html version is good for the duty?

Reply:
Hmtl looks good, however now it only shows 4 tests, but maybe it only shows the files and not the test itself